apps/frontend/src/components/global/ExportCKLModal.vue

```diff
@@ -13,28 +13,191 @@
+                            >
+                              <v-card
+                                :color="active ? 'primary' : ''"
+                                @click="toggle"
+                              >
+                                <v-card-title>ID</v-card-title>
+                                <v-card-text
+                                  >Example of ID:
+                                  {{ file.idexample }}</v-card-text
+                                >
+                              </v-card>
+                            </v-item>
+                            <v-item
+                              v-slot="{active, toggle}"
+                              value="gid"
+                              class="flex-grow-1"
+                            >
+                              <v-card
+                                :color="active ? 'primary' : ''"
+                                @click="toggle"
+                              >
+                                <v-card-title>GID</v-card-title>
+                                <v-card-text
+                                  >Example of GID:
+                                  {{ file.gidexample }}</v-card-text
+                                >
+                              </v-card>
+                            </v-item>
+                          </v-row>
+                        </v-item-group>
+                      </v-card-text>
+                    </v-card>
+                  </v-row>
+                  <v-row>
+                    <div
+                      v-for="(profile, profileIndex) in file.profiles"
+                      :key="profileIndex"
+                    >
+                      <v-card>
+                        <v-card-title>{{ profile.extractedname }}</v-card-title>
+                        <v-card-text>
+                          <v-text-field
+                            v-model="profile.title"
+                            label="Name"
+                            :placeholder="profile.titleplaceholder"
+                          />
+                          <v-text-field
+                            v-model="profile.version"
+                            label="Version"
+                            type="number"
+                            :placeholder="profile.versionplaceholder"
+                          />
+                          <v-text-field
+                            v-model="profile.releasenumber"
+                            label="Release Number"
+                            type="number"
+                            :placeholder="profile.releasenumberplaceholder"
+                          />
+                          <label for="release-date-datepicker">
+                            Release Date
+                          </label>
+                          <v-date-picker
+                            id="release-date-datepicker"
+                            v-model="profile.releasedate"
```

Comment:
Is there a way to have the year attached to the calendar and not populated unless picked on the left side? Because it only updates the year when selecting a date if I am scrolling through the months. Can be in v2 if that will be time consuming.

Reply:
I'm not 100% sure what you mean.  It says the current year there since the calendar needs a date to be centered around on initialization.  If you export the ckl file, it doesn't actually populate any values so the releaseinfo string is still just the release number.
![image](https://github.com/mitre/heimdall2/assets/2015821/c1680d50-5f4d-4e32-a502-e83f34be587d)
^ this is from exporting the rhel7 baseline in samples with no modifications in the export modal